src/status_im/chat/views/toolbar_content.cljs

```diff
@@ -36,29 +34,37 @@
                           (* 100)
                           (.round js/Math)))]
 
-    (str (label :t/sync-in-progress) " " percentage "% " currentBlock)))
+    (str (i18n/label :t/sync-in-progress) " " percentage "% " currentBlock)))
+
+(defn last-activity-syncing [sync-state]
+  (letsubs [state [:get :sync-data]]
+    [react/text {:style styles.screen/last-activity-text}
+     (case sync-state
+       :in-progress (in-progress-text state)
+       :synced (i18n/label :t/sync-synced)
+       nil)]))
```

Comment:
should it be `defview` instead of `defn`?

Reply:
Good catch. The really weird thing is that this worked